### PR TITLE
Convert coordinates

### PR DESCRIPTION
**Pull request by realtalin**
dec to dms conversion implemented, need to change the way the fields (dec and dms) affect eachother to add conversion the other way around

### CODE REVIEW

**Comment by realtalin** [comment]
conversion both ways now implemented, it's convoluted enough that i want to write some tests for it before merging to main

File: frontend/src/util/coordinateConversion.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// get decimal part by converting to string to avoid false precision
+const getDecimalPart = (value: number) => {
+  const stringValue = String(value)
+  const index = stringValue.indexOf('.')
+  return Number('0.' + (index > -1 ? stringValue.substring(index + 1) : '0'))
+}
+
+export const convertDecToDms = (dec: number | null | undefined, isLatitude: boolean) => {
```
Comment (iritmaximus) [inline frontend/src/util/coordinateConversion.ts:8]: Could it be clearer for `isLatitude` to be a string type that accepts `"latitude"` or `"longitude"`? It is a bit unclear what the boolean means when calling the function: `convertDecToDms(<some number>, true)`.

Comment (realtalin) [reply frontend/src/util/coordinateConversion.ts:8]: true, will change

**Review by iritmaximus — APPROVED** [review]
LGTM

**Pull request merged**